layers/dns_test.go

```diff
@@ -576,10 +576,8 @@ var testMalformedRootQuery = []byte{
 
 func TestMalformedRootQuery(t *testing.T) {
```

Comment:
Could you rename the test. It still suggests the packet is malformed although it isnt.

Reply:
Changed to TestBlankNameRootQuery.